video_info.gemspec

```
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.test_files   = s.files.grep(%r{^spec/})
   s.require_path = 'lib'
 
-  s.required_ruby_version = '>= 2.2.0'
+  s.required_ruby_version = '> 2.4.0'
```

What about that instead? I find it easier to parse.

```suggestion
  s.required_ruby_version = '>= 2.5.0'
```

---

Sure! Looks better